src/utils/createNoStringParameterToFunctionWalker.ts

```diff
@@ -114,8 +114,13 @@ export function createNoStringParameterToFunctionWalker(
     function getFunctionTargetType(expression: ts.CallExpression): string | undefined {
         if (expression.expression.kind === ts.SyntaxKind.PropertyAccessExpression && typeChecker) {
             const propExp: ts.PropertyAccessExpression = <ts.PropertyAccessExpression>expression.expression;
-            const targetType: ts.Type = typeChecker.getTypeAtLocation(propExp.expression);
-            return typeChecker.typeToString(targetType);
+            try {
```

Comment:
This change generally LGTM, but let's wrap all the new `try`/`catch` locations in a single method with a comment including a reference to `https://github.com/microsoft/tslint-microsoft-contrib/issues/859`. That'll help explain it in the code.

Reply:
Added comments with detailed description and link to each `try...catch` catch location